jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/FabTests.java

```diff
@@ -17,14 +17,4 @@ public void defaultFabTest() {
         fabFrame.fab.click();
         fabFrame.fab.is().enabled();
     }
-
-    @Test(expectedExceptions = {RuntimeException.class},
-            expectedExceptionsMessageRegExp = ".*(Can't perform click. Element is disabled)")
-    public void disabledFabTest() {
-        inputFabDisabledPage.open();
-
-        fabFrame.fab.is().disabled();
-        fabFrame.fab.click();
-    }
-
```

Comment:
Without this test no check for disabled fab at all

Reply:
Runtime errors processed in core